src/mem/allocconfig.h

```diff
@@ -4,6 +4,34 @@
 
 namespace snmalloc
 {
+  void error(const char* const str);
```

Comment:
Not sure why this is here?  Is this actually used? @rschust I know this was there before, but not sure we want this prototype anywhere now?

Reply:
It is being used by alloc.h, globalalloc.h and malloc.cc, but the forward declaration should (and will) be on pal.h.

Reply:
Okay. Thanks